src/lib/assets/CopyIcon.svelte

```diff
@@ -1,11 +1,15 @@
 <script lang="ts">
-  export let fillColor = '#fff';
-  export let handleClick;
-  export let handleMouseEnter;
-  export let handleMouseLeave;
+  interface Props {
```

Comment:
I know why they did this change in Svelte 5, but it is an annoying change for TypeScript types.